SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServiceSettings.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
```

Comment:
nice! this should make it much easier to have some settings that can be used by all modules